eng/Versions.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VersionPrefix>0.1.2</VersionPrefix>
+    <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
+    <UsingToolXliff>false</UsingToolXliff>
```

Comment:
NIT: I'm not sure if UsingToolXliff should go in this file, this feels more like something that should go in Directory.Build.props.